src/diffusers/pipelines/wuerstchen/diffnext.py

```diff
@@ -0,0 +1,216 @@
+import math
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from ...configuration_utils import ConfigMixin, register_to_config
+from ...models.modeling_utils import ModelMixin
+from .common import AttnBlock, LayerNorm2d, ResBlockStageB, TimestepBlock
+
+
+class DiffNeXt(ModelMixin, ConfigMixin):
+    @register_to_config
+    def __init__(
+        self,
+        c_in=4,
+        c_out=4,
+        c_r=64,
+        patch_size=2,
+        c_cond=1024,
+        c_hidden=[320, 640, 1280, 1280],
+        nhead=[-1, 10, 20, 20],
+        blocks=[4, 4, 14, 4],
+        level_config=["CT", "CTA", "CTA", "CTA"],
+        inject_effnet=[False, True, True, True],
+        effnet_embd=16,
+        clip_embd=1024,
+        kernel_size=3,
+        dropout=0.1,
+        self_attn=True,
```

Comment:
Is this always `True` - can we maybe remove it then from the config?

Reply:
Might not be in future versions.

Reply:
We should still remove it then, we should not write code for future versions because we don't know what future versions look like. We can always later add it, but for now it would be dead code which is not great

Reply:
will remove ok